src/extent.c

```diff
@@ -1255,7 +1255,7 @@ extent_alloc_default(extent_hooks_t *extent_hooks, void *new_addr, size_t size,
 	assert(arena != NULL);
 
 	return extent_alloc_default_impl(tsdn, arena, new_addr, size,
-	    alignment, zero, commit);
+	    ALIGNMENT_CEILING(alignment, PAGE), zero, commit);
```

Comment:
I.e. PAGE_CEILING?

Reply:
That was my initial thought as well, but the semantics are different, even if the result is the same for valid inputs.  `PAGE_CEILING` rounds up to the nearest multiple of the page size, whereas `ALIGNMENT_CEILING` chooses the greater of two powers of two.